### DISCUSSION TITLE
Pause after first print with Octolapse

### DISCUSSION THREAD

**Discussion by acheljay** [smartin015/continuousprint#212] [General]
Hi, we have a cr-30 and we are using continuous print with success except that we have add octolapse to generate time lapse of the batch print ... if octolapse is active, it makes the timelapse of the first print but put in pause the queue at the second cause it don't find the position of the head ... is it possible to avoid problems with octolapse and conveyor belt queue ?

Many thks

**Comment by smartin015** [comment]
Hey @acheljay - I'd love to help with this, but I don't have enough details. Can you please open an issue and include a sysinfo bundle? Thanks.

**Reply by acheljay** [reply]
Hi Martin, I think I have found where the problem come from, but not sure (it's printing for a test). From what I saw, its a problem of finding the origin of the print head at the end of the print when the bed cool down before restarting another print from the queue, so I reconfigure the octolapse printer preset based on creality specs of the CR-30 and force the origin ... my test print take 5h for the first print of the queue so if there is always a problem after the first print I will repost here and if the problem is resolved I will post how I managed it for the CR-30 (could be cool to have a timelapse of the whole queue than just the timelapse of the first onbject and be obliged to relaunch the queue) ;) .

**Reply by acheljay** [reply]
how could I upload here the sysinfo bundle ? never done it before ...

**Comment by acheljay** [comment]
I think that I found the problem ... so the print didn't stop, logically octolapse took all the frames BUT it fullfilled all the space on the pi sd card (and the pi don't want to start until I clean space ... need to plug it on a linux computer to do it).

Logically the problem came from the configuration of the CR-30 and its origin coordinate for the snapshot, I will post the options of octolapse when I recover an functionnal octoprint ;)

**Reply by smartin015** [reply]
Thanks for the update - it sounds like you're finding a solution so a sysinfo isn't necessary, but so you know in the future: bug reports are done by clicking `Issues > New Issue > Bug Report` ([here's](https://github.com/smartin015/continuousprint/issues/new?assignees=&labels=bug&template=bug_report.md&title=)) a quick link). That will direct you on how to create a sysinfo bundle.
